src/components/WorkersLateIndicator.vue

```diff
@@ -16,20 +16,41 @@
   import { useSubscription } from '@prefecthq/vue-compositions'
   import { computed, toRefs } from 'vue'
   import { useWorkspaceApi } from '@/compositions'
+  import { UnionFilters } from '@/types'
   import { toPluralString } from '@/utilities'
 
   const props = defineProps<{
     workPoolName: string,
     workPoolQueueNames?: string[],
   }>()
 
-  const workPoolQueueNames = computed(() => props.workPoolQueueNames ?? [])
 
   const api = useWorkspaceApi()
+
   const { workPoolName } = toRefs(props)
+  const workPoolQueueNames = computed(() => props.workPoolQueueNames ?? [])
```

Comment:
Not sure you need this computed property? Could just use `props.workPoolQueueNames` directly on like 45, no?

Reply:
Good catch!